src/chapters/sql-part2/exercises.rst

```diff
@@ -0,0 +1,107 @@
+Exercises: SQL, Part 2
+=======================
+
+In the :ref:`previous chapter <sql-part1-exercises>`, we set up a database to store information about seeds. Let's expand that idea now that we know all about foreign keys!
+
+Create a New Schema
+-------------------
+
+To get started, let's set up a :ref:`new schema <sql-part1-exercises>` in MySQL Workbench and call it ``garden``.
+In the last chapter, we learned we can do this by clicking on *New Schema*, naming it ``garden``, accepting the defaults, and clicking *Apply*.
+
+Import New Tables
+-----------------
+
+The starter data for these exercises is in the `Github repo <https://github.com/LaunchCodeEducation/java-web-development-starter-data>`_ under ``lesson-15-exercises``.
+To import the data in our ``.csv`` files, we need to create three empty tables. 
+
+.. admonition:: Note
+
+   The instructions state that you should be creating each table in separate query tabs. 
+   If it works better for you and your workflow, feel free to create the three tables in one query tab!
+
+The ``plant`` Table
+^^^^^^^^^^^^^^^^^^^
+
+Our first table is ``plant``. ``plant`` needs to have four columns:
+
+#. ``plant_id``, which will be the primary key for our database.
+#. ``plant_name``, which will contain the name of the plant.
+#. ``zone`` is an integer for the hardiness zone where our plant will live in best.
+#. ``season`` is a string that specifies the season that the plant blooms in.
+
+In a new query tab, use ``CREATE TABLE`` to make the ``plant`` table.
+The data for the ``plant`` table is in the ``plants.csv`` file. You can download it directly from Github.
+
+The ``seeds`` Table
+^^^^^^^^^^^^^^^^^^^
+
+Our second table is ``seeds``. ``seeds`` needs to have five columns:
+
+#. ``seed_id``, which will auto increment.
+#. ``expiration_date``, which will be of the ``DATE`` type.
+#. ``quantity`` is an integer for the number of seeds we have available.
+#. ``reorder`` of type ``BOOL`` and tells us if we need to reorder that type of seed.
+#. ``plant_id`` is the foreign key that connects ``seeds`` to ``plant``.
+
+In a new query tab, use ``CREATE TABLE`` to make the ``seeds`` table.
+The data for the ``seeds`` table is in the ``seeds.csv`` file. You can download it directly from Github.
+
+The ``garden_bed`` Table
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Our third table is ``garden_bed``. ``garden_bed`` needs to have four columns:
+
+#. ``space_number``, which will increment as we plant new plants.
+#. ``date_planted`` of type ``DATE`` for the day that we planted the plant on.
```

Comment:
To reduce "plant" and remove ending preposition:
"...as we sow new plants."
"...for the day we started the new plant."

Reply:
Done!